### PR TITLE
FSC test network

### PR DESCRIPTION
**Pull request by mbrandenburger**
This PR adds a new playground for FPC Chaincode. We introduce a "single command" which deploys a local "for-testing-only-purpose" Fabric Network and installs a FPC chaincode, including all lifecycle interactions. The goal is to provide an easy and quick way to play with your FPC chaincode without becoming an expert of deploying a test network. To implement this new playground, we use FSC to specify and run a Fabric network,

A readme walks the through the process of setting up the network and interact with the FPC chaincode using our simple-cli app.  

Signed-off-by: Marcus Brandenburger <bur@zurich.ibm.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing changes and/or breaks backward compatability?**:

```

### CODE REVIEW

**Review by bvavala — COMMENTED** [review]
lgtm

**Comment by ikegawa-koshi** [comment]
Nice work!

File: samples/deployment/fabric-smart-client/the-simple-testing-network/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+export CC_PATH=$FPC_PATH/samples/chaincode/echo
+make -C $CC_PATH
+make -C $FPC_PATH/ecc DOCKER_IMAGE=fpc/fpc-${CC_NAME}$${HW_EXTENSION} DOCKER_ENCLAVE_SO_PATH=$CC_PATH/_build/lib all docker
+``` 
+
+Download fabric binaries:
+
+```bash
+curl -sSL https://bit.ly/2ysbOFE | bash -s -- 2.2.7 1.4.9 -s -d
+export FAB_BINS=$(PWD)/bin
```
Comment (ikegawa-koshi) [inline samples/deployment/fabric-smart-client/the-simple-testing-network/README.md:32]: In the Fpc-development container, $(pwd) had to be lower case.
`export FAB_BINS=$(pwd)/bin`

**Review by ikegawa-koshi — COMMENTED** [review]
I just checked on my machine, In the Fpc-development container, $(pwd) had to be lower case.

File: samples/deployment/fabric-smart-client/the-simple-testing-network/README.md (hunk 1 above)
Comment (mbrandenburger) [reply samples/deployment/fabric-smart-client/the-simple-testing-network/README.md]: thanks! good catch!

**Pull request merged**